pallets/foreign-investments/src/lib.rs

```diff
@@ -248,7 +248,7 @@ pub mod pallet {
 	///
 	/// NOTE: The storage is killed when the swap order no longer exists
 	#[pallet::storage]
-	pub(super) type ForeignIdToSwapId<T: Config> =
+	pub type ForeignIdToSwapId<T: Config> =
```

Comment:
NIT. In the `swaps.rs` module, I've recently added a function `foreign_id_from()` to avoid exposing the storage and generate the correct error in case of wrong usage.

Reply:
Awesome, did not catch that.

Reply:
Needed the reverse direction, added a getter here: [6b88c4b](https://github.com/centrifuge/centrifuge-chain/pull/1700/commits/6b88c4be797fcc4022fccc3366d93a9959ffb40a)